src/validator/validators.js

```diff
@@ -200,7 +198,7 @@ var register = function registerValidator(name, validator, force) {
     if (!_.isObject(validator) || !_.isString(validator.message) || !_.isFunction(validator.validate)) {
         throw new Error(
             'A validator must be an object with a message and a validate method, but given : ' +
-            JSON.stringify(validator)
+                JSON.stringify(validator)
```

Comment:
**chore:** Should not this be converted to a string literal?

Reply:
Thanks, done